src/test/kotlin/com/zepben/evolve/database/sqlite/cim/upgrade/changesets/network/ChangeSet55NetworkValidator.kt

```diff
@@ -0,0 +1,87 @@
+        "INSERT INTO curve_data (curve_mrid, x_value, y1_value, y2_value, y3_value) VALUES ('mrid', 1.0, 1.0, 1.0, 1.0);",
+        "INSERT INTO grounding_impedances (mrid, name, description, num_diagram_objects, location_mrid, num_controls, normally_in_service, in_service, commissioned_date, base_voltage_mrid, r, x) VALUES ('mrid', 'name', 'description', 1, 'location_mrid', 1, true, true, 'commissioned_date', 'base_voltage_mrid', 2.0, 2.0);",
+        "INSERT INTO petersen_coils (mrid, name, description, num_diagram_objects, location_mrid, num_controls, normally_in_service, in_service, commissioned_date, base_voltage_mrid, r, x_ground_nominal) VALUES ('mrid', 'name', 'description', 1, 'location_mrid', 1, true, true, 'commissioned_date', 'base_voltage_mrid', 3.0, 3.0);",
+        "INSERT INTO reactive_capability_curves (mrid, name, description, num_diagram_objects) VALUES ('mrid', 'name', 'description', 1);",
+        "INSERT INTO synchronous_machines (mrid, name, description, num_diagram_objects, location_mrid, num_controls, normally_in_service, in_service, commissioned_date, base_voltage_mrid, control_enabled, regulating_control_mrid, rated_power_factor, rated_s, rated_u, p, q, base_q, condenser_p, earthing, earthing_star_point_r, earthing_star_point_x, ikk, max_q, max_u, min_q, min_u, mu, r, r0, r2, sat_direct_subtrans_x, sat_direct_sync_x, sat_direct_trans_x, x0, x2, type, operating_mode) VALUES ('mrid', 'name', 'description', 1, 'location_mrid', 1, true, true, 'commissioned_date', 'base_voltage_mrid', true, 'regulating_control_mrid', 1.0, 1.0, 1, 1.0, 1.0, 1.0, 1, true, 1.0, 1.0, 1.0, 1.0, 1, 1.0, 1, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 'generator', 'generator');",
+        "INSERT INTO synchronous_machines_reactive_capability_curves (synchronous_machine_mrid, reactive_capability_curve_mrid) VALUES ('synchronous_machine_mrid', 'reactive_capability_curve_mrid');"
+    )
+
+    override fun validateChanges(statement: Statement) {
+        ensureNewAdded(statement)
+        validateRows(statement, "SELECT * FROM usage_points",
+            { rs ->
+                assertThat(rs.getString("mrid"), equalTo("mrid"))
+                assertThat(rs.getString("name"), equalTo("name"))
+                assertThat(rs.getString("description"), equalTo("desc"))
+                assertThat(rs.getInt("num_diagram_objects"), equalTo(0))
+                assertThat(rs.getString("location_mrid"), equalTo("l_id"))
+                assertThat(rs.getBoolean("is_virtual"), equalTo(true))
+                assertThat(rs.getString("connection_category"), equalTo("connection_category"))
+                assertThat(rs.getInt("rated_power"), equalTo(2))
+                assertThat(rs.getInt("approved_inverter_capacity"), equalTo(2))
+                assertThat(rs.getString("phase_code"), equalTo("AN"))
+            }
+        )
```

Comment:
suggestion: Recommend that you split the `ensureNewAdded` into a call for each added table and move the `usage_points` validation into its own function such that you can see that each table used in the changeset is being tested.

e.g.
```
`ensure modified usage_points`()

`ensure added curve_data`()
`ensure added grounding_impedances`()
`ensure added petersen_coils`()
`ensure added reactive_capability_curves`()
`ensure added synchronous_machines`()
`ensure added synchronous_machines_reactive_capability_curves`()
```

Reply:
Updated